### PR TITLE
Bug 1879533: oVirt UPI - Better explanation of the inventory

### PR DESCRIPTION
**Pull request by gekorob**
Adds a working inventory.yml example and better explanations to that

### CODE REVIEW

File: docs/user/ovirt/install_upi.md [hunk 1]
```diff
@@ -211,14 +327,34 @@ storage domains for masters and workers. This might be useful as etcd key-value
...
+        network: lab
+        profile: lab
 ```
 
+The user can customize the parameters of both profiles according to the needs and the minimum requirements.
+
+* `cluster`: it's already set according to the value of the variable in the [General Section](#general-section).
+* `memory, sockets, cores`: mandatory parameters necessary to define the common specs of the VMs generated from this profile.
+* `template`: name of the template that the virtual machine will be based on (refer to [this](#create-templates-and-vms) for further information about templates).
+* `operating_system`: sets the vm OS type. With RHV 4.4 it's mandatory to use the value `rhcos_x64` to allow the `ignition script`
```
Comment (jan-zmeskal) [inline docs/user/ovirt/install_upi.md:346]: I think in upstream docs we should `s/RHV/oVirt/`

**Review by jan-zmeskal — APPROVED** [review]

File: docs/user/ovirt/install_upi.md (hunk 1 above)
Comment (gekorob) [reply docs/user/ovirt/install_upi.md:346]: Agree, I'll change it

Comment (gekorob) [reply docs/user/ovirt/install_upi.md:346]: I'll change it from RHV to oVirt/RHV like it is also in the IPI docs

**Comment by Gal-Zaidman** [comment]
@jan-zmeskal @gekorob is this PR ready to merge ? I see you haven't resolved the conversation with @jan-zmeskal

**Comment by jan-zmeskal** [comment]
@Gal-Zaidman I checked it and I believe it's ready

**Pull request merged**